### PR TITLE
Add step to check if `pyenv` is usable by the user

### LINKED ISSUE #4448: Add steps to update `pyenv` path
https://github.com/Scille/parsec-cloud/blob/10e8e907339f57a3ef66013adf727010aea4ec23/docs/development/quickstart.md?plain=1#L67-L68

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
i.e.: Add a step that indicate the user to modify it's `PATH` variable to add the installation path of `pyenv`

Closes #4448

### CODE REVIEW

**Review by touilleMan — APPROVED** [review]
:+1:

File: docs/development/quickstart.md [hunk 1]
```diff
@@ -61,14 +61,37 @@ To start hacking, follow the basic steps detailed below:
...
+
+       2. Check your `pyenv` installation:
+
+          If the following command work, you can skip this step
+
+          ```shell
+          pyenv --version
+          ```
+
+          If this command failed with `command not found` (or alike) this mean your OS can't find `pyenv` inside our `PATH` variable.
```
Comment (mmmarcos) [inline docs/development/quickstart.md:83]:
```suggestion
          If this command failed with `command not found` (or alike) this means your OS can't find `pyenv` inside your `PATH` variable.
```

File: docs/development/quickstart.md [hunk 2]
```diff
@@ -61,14 +61,37 @@ To start hacking, follow the basic steps detailed below:
...
+
+          To correct that you need to update your `PATH` variable, if you've installed `pyenv` via the step above you mostly need to add `$HOME/.pyenv/bin` to that variable.
+
+          > `$HOME` is you home path.
+
+       3. Install the correct python version using `pyenv`
+
+          ```shell
+          # Compile Python
```
Comment (mmmarcos) [inline docs/development/quickstart.md:92]:
```suggestion
          # Install a specific Python version
```

File: docs/development/quickstart.md [hunk 3]
```diff
@@ -61,14 +61,37 @@ To start hacking, follow the basic steps detailed below:
...
+
+          If the following command work, you can skip this step
+
+          ```shell
+          pyenv --version
+          ```
+
+          If this command failed with `command not found` (or alike) this mean your OS can't find `pyenv` inside our `PATH` variable.
+
+          To correct that you need to update your `PATH` variable, if you've installed `pyenv` via the step above you mostly need to add `$HOME/.pyenv/bin` to that variable.
```
Comment (mmmarcos) [inline docs/development/quickstart.md:85]:
```suggestion
          To correct that you need to update your `PATH` variable. If you've installed `pyenv` via the step above you need to add `$HOME/.pyenv/bin` to that variable.
```

File: docs/development/quickstart.md [hunk 4]
```diff
@@ -61,14 +61,37 @@ To start hacking, follow the basic steps detailed below:
...
+
+          > For more installation methods for `pyenv`, see <https://github.com/pyenv/pyenv#installation>
+
+       2. Check your `pyenv` installation:
+
+          If the following command work, you can skip this step
+
+          ```shell
+          pyenv --version
+          ```
```
Comment (mmmarcos) [inline docs/development/quickstart.md:81]:
```suggestion
          ```shell
          pyenv --version
          ```

          If the previous command worked, you can skip directly to step 3.
```

**Review by mmmarcos — APPROVED** [review]
:+1:

**Pull request merged**